### PR TITLE
yo something to add

### PR DESCRIPTION
**Pull request by NotEnoughSpeed**

### CODE REVIEW

**Comment by NotEnoughSpeed** [comment]
yo

**Review by msprojecter — COMMENTED** [review]
this change looks nice

**Review by msprojecter — CHANGES_REQUESTED** [review]
I'd like you to change it a bit then we're cool

File: reallycoolfile [hunk 1]
```diff
@@ -1 +1,3 @@
 something cool
+
+I'd like to add
```
Comment (msprojecter) [inline reallycoolfile:3]: this look nice

File: reallycoolfile [hunk 2]
```diff
@@ -1 +1,3 @@
 something cool
+
```
Comment (msprojecter) [inline reallycoolfile:2]: this looks unnecessary

**Review by msprojecter — COMMENTED** [review]
yes

File: reallycoolfile (hunk 2 above)
Comment (NotEnoughSpeed) [reply reallycoolfile:2]: I think I can fix that

File: reallycoolfile (hunk 1 above)
Comment (NotEnoughSpeed) [reply reallycoolfile:3]: thanks <3

File: reallycoolfile [hunk 3]
```diff
@@ -1,3 +1,2 @@
 something cool
-
 I'd like to add
```
Comment (msprojecter) [inline reallycoolfile:2]: this can use some more love

Comment (NotEnoughSpeed) [reply reallycoolfile:2]: ok sure

**Pull request merged**